.github/workflows/test.yml

```diff
@@ -557,7 +573,13 @@ jobs:
       cancel-in-progress: true
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Repository
+        uses: actions/checkout@v4
+      
+      - name: Node
```

Comment:
No - this was something discussed between @AlexBramhill and I about the node version in e2e tests. All these steps do is ensure that we are using the same node version here as we are for the rest of our tests rather than using the bundled version from cypress